.circleci/export-playwright-env.sh

```diff
@@ -9,6 +9,12 @@ echo "export SITE_PASSWORD=$sitePwd" >> playwright-env/envvars
 export bspToken=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.bsp | .[] | select(.env==\"$ENV\") | .token")
 echo "export BSP_TOKEN=$bspToken" >> playwright-env/envvars
 
+export pubsubTopicName=$(vault read --format=json secret/pepper/test/v1/e2e | jq -r ".data.mercury | .[] | select(.env==\"$ENV\") | .pubsubTopicName")
```

Comment:
It looks like we aren't dipping into pubsub for these tests.  Do we need these values for testing?

Reply:
Can you say a bit more about why we need to interact with pubsub directly when there's a "place order" button on the UI?

Reply:
yeah - the "place order" button usually just fills the Latest Sequencing Order Date column and the Latest Order Number column. I need to use pubsub in order to make sure that the Latest Order Status and Latest PDO Number are also filled out. 
Below is state or orders with just "place order" selected (see the ones with "10/15/2024" as the Latest Sequencing Order Date):
<img width="1082" alt="Screenshot 2024-10-15 at 9 51 11 AM" src="https://github.com/user-attachments/assets/2b2b44f8-82df-4b9e-9ee8-96a0c377860f">

Below is after using pubsub (Latest Order Status and Latest PDO Number are now filled out):
<img width="1082" alt="Screenshot 2024-10-15 at 9 53 36 AM" src="https://github.com/user-attachments/assets/1e4c580e-cfc4-465b-9a23-d6ad5fdd96b6">

attaching the doc that has the testing steps since video reference is too big to attach: [https://docs.google.com/document/d/1TGCnJGNz-HPYo1zO7wEeVHjkhm0NEgOEQHwyoCU12aI/edit?tab=t.0](url)